drake/doc/fedora.rst

```diff
@@ -0,0 +1,11 @@
+*************************************************
+Centos / Red Hat Enterprise Linux 7.2 / Fedora 23
```

Comment:
I think we should list this as (UNSUPPORTED) by annotating the title of this file.